HETG_CHECK/HETG_Check.py

```diff
@@ -0,0 +1,283 @@
+#! /usr/local/bin/python
+#
+# HETG_Check.py - Perigee Passage Gratings Insertion Check
+#
+"""
+
+Guideline: 
+https://occweb.cfa.harvard.edu/twiki/bin/view/Constraints/MPGuidelines/Development/GuidelineRadiationSafingWithoutHRC
+
+" When HRC data is not used for onboard radiation monitoring and safing, ensure the following:
+
+    HETG is inserted by rad entry, either during the same maneuver used to safe the 
+    SIM or prior, and kept in until at least the start of the maneuver to the first 
+    target exiting the rad zone Priority 2"
+   
+Check the Weekly load to be sure that the HETG has been inserted for the Perigee passage.
+
+IMPORTANT: VO (Vehicle Only) loads DO NOT HAVE OORMPDS or OORMPEN
+              VO loads DO have EQF013M and EEF1000 and XQF013M, XEF1000
+           VpS (Vehicle plus Science) loads have EQF013M and EEF1000 and XQF013M, XEF1000
+                            
+           So since the proton and electron events exist in both VO and VpS files, this 
+           program focuses on using EQF013M, EEF1000, XQF013M and XEF1000 to determine
+           Radzone entry for all loads.
+
+  There are 4 values you need in order to be sure the Gratings are in:
+
+    HETG In  - MSID= 4OHETGIN
+    HETG Out - MSID= 4OHETGRE
+    Radzone Entry
+    Radzone Exit
+
+    In this progam these 4 commands or states will be referred to as HETG states.
+    The complete set of 4 states will be referred to as a state group.
+
+  The HETG is not guaranteed to be in prior to the HRC-S SIMTRANS, and it is usually retracted
+  during the outbound ECS measurement so it's retracted prior to XEF1000. Therefore this
+  program will check to be sure it's in during the Perigee Passage and will report the 
+  percentage time of the passage it was in.
+
+"""
+import argparse
+import numpy as np
+import sys
+
+# Import the BackstopHistory class
+from backstop_history import BackstopHistory
+
+# ACIS Ops Imports
+import apt_date_secs as apt
+import Calc_Delta as cd
+import ORP_File_Class as ofc
+
+exec(open('/home/gregg/LR_CODE/UTILITIES/OFLS_File_Utilities.py').read())
```

Comment:
Is there a different way to do this? I realize you asked me about this recently and I don't think I answered. Could we just put the commands you need from this file into a file in this repository?

Reply:
Ok, now I see that this is in this repository. But we should be able to do some kind of import instead of `exec`.

Reply:
Oversight on my part. I intended to change that to an import once initial coding and testing were done.
Fixed and tested